spec/features/recipes_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe "recipes show" do
+
+    before(:each) do
+        @recipe_1 = Recipe.create!(name: "Hamburger", complexity: 1, genre: "easy")
+        @recipe_2 = Recipe.create!(name: "Fries", complexity: 2, genre: "fried")
+        @ingredient_1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+        @ingredient_2 = Ingredient.create!(name: "Salt", cost: 4)
+        @ingredient_3 = Ingredient.create!(name: "Potato", cost: 5)
+
+        @recipe_1.ingredients << [@ingredient_1]
+        @recipe_2.ingredients << [@ingredient_2, @ingredient_3]
+    end
+
+    # User Story 2
+    describe "when I visit recipies show page I see a recipies ingredients" do
+       it "displays the recipe's name, complexity, genre, and ingredient names" do
+        
+            visit "/recipes/#{@recipe_2.id}"
+
+            expect(page).to have_content(@recipe_2.name)
+            expect(page).to have_content(@recipe_2.complexity)
+            expect(page).to have_content(@recipe_2.genre)
+
+            expect(page).to have_content(@ingredient_2.name)
+            expect(page).to_not have_content(@ingredient_1.name)
+       end
+    end
```

Comment:
One thing I want to call out is making your tests more robust with anchor text. Anchor text is text that anchors something to the page. So instead of just `expect(page).to have_content(@recipe_2.complexity)`, you do something like `expect(page).to have_content("Complexity: #{@recipe_2.complexity}")`.

This reduces the chance of false positives, so instead of just saying, find `2` on the page, we are saying find `Complexity: 2` on the page.